core/gui/src/app/workspace/component/workspace.component.ts

```diff
@@ -24,7 +24,7 @@ import { WorkflowConsoleService } from "../service/workflow-console/workflow-con
 import { OperatorReuseCacheStatusService } from "../service/workflow-status/operator-reuse-cache-status.service";
 import { CodeEditorService } from "../service/code-editor/code-editor.service";
 
-export const SAVE_DEBOUNCE_TIME_IN_MS = 300;
+export const SAVE_DEBOUNCE_TIME_IN_MS = 5000;
```

Comment:
I only see a change in debounce time for persisting workflow. can we also turn the version creation "time interval" to be larger?